app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        render json: notes_filtered, status: :ok, each_serializer: IndexNoteSerializer
+      end
+
+      def show
+        render json: show_note, status: :ok, serializer: ShowNoteSerializer
```

Comment:
En estos casos solemos llamar al método directamente `note`. Nos permitiría reutilizar el método en algún otro servicio, sin que sea confuso 
```suggestion
        render json: note, status: :ok, serializer: ShowNoteSerializer
```